doc/source/_toc.yml

```diff
@@ -379,17 +380,21 @@ parts:
               sections:
               - file: cluster/vms/examples/ml-example
             - file: cluster/vms/references/index
+          - file: cluster/configure-manage-dashboard
           - file: cluster/running-applications/index
             title: Applications Guide
           - file: cluster/faq
           - file: cluster/package-overview
 
-      - file: ray-observability/monitoring-debugging/monitoring-debugging
+      - file: ray-observability/index
         title: "Monitoring and Debugging"
         sections:
+          - file: ray-observability/getting-started
+          - file: ray-observability/key-concepts
           - file: ray-observability/user-guides/index
             title: User Guides
             sections:
+              - file: ray-observability/user-guides/general-troubleshoot
```

Comment:
What is this for?

Reply:
That is the old "Gotchas" page. I think we said we would remove it, but it looked like it might be useful, so I found a place an title for it. I can easily remove it if it's not useful.

Reply:
Got it. The content itself is useful! However, it _can_ belong to "Troubleshoot applications" or maybe they should live in ray core or other sections...

Reply:
Let's not put it in Ray Core because users of the libraries will not know to look there. I can put it in `Troubleshoot applications` for now.

Reply:
> I can put it in Troubleshoot applications for now.

SG.

> Let's not put it in Ray Core because users of the libraries will not know to look there.

However, we mostly use core in those examples. Not sure how useful it is for library users. We probably need to improve them and make it more relatable to library users.